test/references/string/string5.txt

```diff
@@ -1,2 +1,2 @@
-This is a

```

Comment:
This was used to check CRLF/LF difference, but given that #64 properly handles this test case, we no longer need it.

The new reference is automatically generated by `rm -rf test/references`.